client/src/components/layout/AddReviewsForm.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react"
+
+const AddReviewsForm = (props) => {
+  const [newReview, setNewReview] = useState({
+    heading: "",
+    description: "",
+    rating: "",
+    userId: "",
```

Comment:
We don't need to track the user id in the form. The express backend is already managing who the signed in user is. Our form should just be submitting data supplied by the user. We do not need so include this userId in the body of the POST fetch request. Instead, when we get to the backend router, we can then access the user through `req.user` as passport is managing the signed in user.